src/ensemble_scheduler/ensemble_scheduler.cc

```diff
@@ -1371,9 +1373,8 @@ EnsembleScheduler::EnsembleScheduler(
   if (Metrics::Enabled()) {
     // Ensemble scheduler doesn't currently support response cache at top level.
     MetricModelReporter::Create(
-        config.name(), 1, METRIC_REPORTER_ID_CPU,
-        false /* response_cache_enabled */, config.metric_tags(),
-        &metric_reporter_);
+        model_id, 1, METRIC_REPORTER_ID_CPU, false /* response_cache_enabled */,
```

Comment:
```suggestion
        model_id, 1 /* model_version */, METRIC_REPORTER_ID_CPU, false /* response_cache_enabled */,
```
I know, that it is not your change, but I would like to elaborate this magic number =)

Reply:
@rmccorm4 could you please clarify why we hardcode `model_version`?

Reply:
Looks like that goes way back to [May 2020](https://github.com/triton-inference-server/server/commit/42fc28f5462ef2c4dd3b4c26d5a13ab315556c35#diff-d2cd2b262fd3de620e660fd32738363c38759afbfb3cd3e6987fcf10b462060d).

If I were to guess, it's because the ensemble model has no `TritonModel*` object associated with it to query a version, and I don't think the model config protobuf object has a version field to query either (like the `config.name()` it uses for model name). There may be a way around it to get some programattic now :shrug: 

Don't think I've ever seen someone raise an issue about this, maybe people don't mess with ensemble versions :sweat_smile:

Reply:
> I know, that it is not your change, but I would like to elaborate this magic number =)

Sure, just adding a comment: [Comment on model version number](https://github.com/triton-inference-server/core/pull/343/commits/b9ec4c5b4dc273f42464e85926b8cb0bc390aca1)